tests/TeamHasPermissionsTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Spatie\Permission\Test;
+
+class TeamHasPermissionsTest extends HasPermissionsTest
+{
+    /** @var bool */
+    protected $hasTeams=true;
+
+
+    /** @test */
+    public function it_can_asign_same_and_diferent_permission_on_same_user_on_diferent_teams()
```

Comment:
"asign" => "assign"
"diferent" => "different"

 (also in other tests/files)

Reply:
ok, i will fix the branch, sorry for my english 😄

Reply:
@drbyte it's done